hack/monitor_prow.py

```diff
@@ -153,5 +158,56 @@ def renderFlavor(pod, dc):
     debug("deployment/{}: pod/{}: got flavor {}".format(dc, pod, lines))
     return lines
 
+
+###Copied from
+###https://docs.google.com/document/d/11Ao_as9Pi2WLIffwn6Xh1XO_857Xra1liAwEVjrGtg8/edit?usp=sharing
+NOT_QUERY = '''
```

Comment:
Why not

```py
blacklist = [
    "Failed to GET .",
    // ...
]
```

Reply:
Because I want the copy/paste to be easy.
Originally, I planned to maintain the query only in one place, here in the script.
When this merges, i will remove the box in the gdoc and link it here.

The current state is somewhere in between but copy/paste is still easy.

If we do blacklist as python list, then we have to compare the log-viewer query with it if the query changes.

Reply:
It changes pretty infrequently and I'd rather not write regex to parse this. Either way you'll need to update two things, let's just make it simple.

Reply:
Done.